TrkHitReco/src/TNTClusterer.cc

```diff
@@ -110,8 +125,9 @@ namespace mu2e
       const auto& chit = chcol[hit.chidx_];
 
       // -- if hit is ok, reassign it right away
-      if (hit.distance_ < dhit_) {
+      if (hit.distance_ < dhit_ || hit.dchi2_ < dhit_) {
```

Comment:
This statement makes the choice of distance scale appear ambiguous.  Can you instead use distance_ in both cases?  then you just cut on 1 variable.

Reply:
Yes, this should work